spyder/widgets/helperwidgets.py

```diff
@@ -455,5 +459,88 @@ def test_msgcheckbox():
     box.exec_()
 
 
+class PanelEmptyWidget(QFrame):
+    """Widget to show a panel/plugin functionality description."""
+
+    def __init__(self, parent, icon_filename, text, description):
+        super().__init__(parent)
+        # Setup widgets
+        # Image
+        image_path = get_image_path(icon_filename)
+        image = QPixmap(image_path)
+        image_label = QLabel(self)
+        image_height = int(image.height())
+        image_width = int(image.width())
+        image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
+                             Qt.SmoothTransformation)
```

Comment:
Does this get pixelated when the scale factor for the app is increased in `Application > Interface > Screen resolution`?

If that's the case, perhaps we should have here a similar code to the one used to load our splash screen.

Reply:
Note: splash screen creation code https://github.com/spyder-ide/spyder/blob/c82d3760e6ca5a2adf87bc0c0b17f48af3dff1ed/spyder/app/utils.py#L184-L204